Rubberduck.Core/UI/UnitTesting/Commands/AddTestModuleCommand.cs

```diff
@@ -131,6 +131,8 @@ private IVBProject GetProject()
                 return activeProject;
             }
 
+            activeProject.Dispose();
```

Comment:
Why do we try to dispose a project here that is wrapping a null reference?

Reply:
This is primarily to remove it from the ComSafe and thus prevent it be GC'd. A `Dispose` on a null-wrapping reference will not actually `Release` the wrapped COM object so it's safe to do.

That said, I really should have fixed the method to use `using` to be more idiomatic.

Reply:
SCWs wrapping a null reference do not get added to the COM safe. So, the dispose here is actually a no-op.

Reply:
Hmm. You're right. Yet I was bought here because from the log, this got GC'd....